docs/governance/tokenomics.md

```diff
@@ -2,200 +2,178 @@
-A validator that has failed to get itself back into the bonded validators set within 21 days is said to be unbonded.
-The number 100 can be updated via a simple governance proposal. – Need testing and verification.
+*Block rewards* =  *staking rewards* + *tx fees* 
+:::
 
-**Only bonded validators can propose or sign blocks.
-Only bonded validators can earn block rewards.
-Only bonded validators’ tokens are subject to slashing.**
+### Staking rewards 
 
-Although the actual act of slashing might occur during the unbonding period.
+Staking rewards occur in a **single pool** **Tx fees** should be the main source of revenue for **validators** and **delegators**. However, due to the expectation of low early engagement numbers on the **Cudos network**, additional rewards are provided to users. and are distributed as follows:  
 
-## Redelegation 
+1. **Bonded validators** (including the **Block proposer**) are assigned rewards proportional to the amount they have staked relative to the entire network. 
+    
+2. Users who have delegated tokens are required to **actively** claim those rewards by sending appropriate txs to the blockchain.
 
-A **delegator** can choose to undelegate their tokens at any time.
+3. The **Block proposer** receives a bonus between 3.67% and 5% of the **total block rewards** depending on the number of validators signing that block.
 
-A **delegator** can redelegate their tokens from **Validator A** to **Validator B** without any penalties or extra waiting times. 
+4. 20% goes to the **community pool**.
 
-To redelegate tokens from **Validator B** to **Validator C** you must wait 21 days. 
+### Distribution Schedule
 
-However, they need to wait for 21 days if they want to further redelegate any amount of tokens from Validator B – irrespective of how much they redelegated to Validator B initially.
+* A total of ~1,5 billion CUDOS tokens will be distributed as staking rewards in 10 years. The exact number of tokens is given by the formula 
 
-A validator can **self-delegate only through the CLI** – explorer v1 doesn’t provide this functionality.
+**f(x) = 1.8 x^2 - 53 x + 358** 
 
-A validator can also have an **additional stake added** to it via the CLI to increase its voter power.
+**where *x* is in years and *f(x)* is in millions of CUDOS.**
 
-## Block Rewards 
+* Hence the amount to be distributed in any given time interval is obtained by integrating the above function over that interval.
 
-The more tokens staked to your Validator node, the higher the likelihood of being chosen to propose and sign a block. 
+* Due to the assumption that tx fees will eventually be the main source of revenue, staking rewards start high and then decrease quadratically in time.
 
-* **Block rewards** are calculated after each block, and is equal to the sum of the **staking rewards** and the **tx fees** for that block.
+### Validator self-delegating
```

Comment:
I suggest we move this section to just above the Staking Rewards section because in the Staking Rewards section we are mentioning being bonded without explaining what it means.

Reply:
done